docs/daisy-reusing-workflows.md

```diff
@@ -0,0 +1,87 @@
+# Reusing Workflows
+
+Instead of requiring you to copy-paste large sections of workflows to perform
+similar tasks, Daisy provides two different step types for workflow reuse:
+`IncludeWorkflow` and `SubWorkflow`. Both step types will cause Daisy to run the
+workflow they specify, but how Daisy does this is different:
+
+### IncludeWorkflow
+
+`IncludeWorkflow` will run the steps of the included workflow in parallel with
+the parent workflow's steps. A workflow included with `IncludeWorkflow` will
```

Comment:
Technically, its not always in parallel unless the dependency map says so. You can also run subWorkflows in parallel the same way. The primary point is that the resource namespaces are shared and therefore it is up to you to sort out collisions. It is useful however for sharing resources between multiple workflows (a good example is a disk that os created in a top level workflow and you want to perform actions on that disk in an included workflow that has common functionality to perform actions on a disk provided as a variable).

Reply:
Right. Thanks for pointing this out. I'll make it clearer that that's the case.